packages/dashboard/src/components/journeysTable.tsx

```diff
@@ -0,0 +1,166 @@
+import { DataGrid, GridColDef } from "@mui/x-data-grid";
+import {
+  CompletionStatus,
+  DeleteJourneyRequest,
+  EmptyResponse,
+  MessageTemplateResourceRequest,
+} from "isomorphic-lib/src/types";
+import { useRouter } from "next/router";
+import React from "react";
+
+import apiRequestHandlerFactory from "../lib/apiRequestHandlerFactory";
+import { useAppStore } from "../lib/appStore";
+import { monospaceCell } from "../lib/datagridCells";
+import DeleteDialog from "./confirmDeleteDialog";
+
+interface Row {
+  id: string;
+  name: string;
+  updatedAt: string;
+}
+
+const baseColumn: Partial<GridColDef<Row>> = {
+  flex: 1,
+  sortable: false,
+  filterable: false,
+  renderCell: monospaceCell,
+};
+
+declare module "@mui/x-data-grid" {
```

Comment:
It looks like this code is duplicated on multiple pages. Is that necessary as well?

Reply:
Nope not necessary. This is a mistake from my end. Sorry once again!